data/get_jeff_data.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import shutil
+import subprocess
+import sys
+import tarfile
+import zipfile
+import glob
+import hashlib
+import argparse
+
+import openmc.data
+
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
+
+thermal_suffix = {20: '01t', 100: '02t', 293: '03t', 296: '03t', 323: '04t',
+                  350: '05t', 373: '06t', 400: '07t', 423: '08t', 473: '09t',
+                  500: '10t', 523: '11t', 573: '12t', 600: '13t', 623: '14t',
+                  643: '15t', 647: '15t', 700: '16t', 773: '17t', 800: '18t',
+                  1000: '19t', 1200: '20t', 1600: '21t', 2000: '22t',
+                  3000: '23t'}
+
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-b', '--batch', action='store_true',
+                    help='supresses standard in')
+args = parser.parse_args()
+
+base_url = 'https://www.oecd-nea.org/dbforms/data/eva/evatapes/jeff_32/Processed/'
```

Comment:
Print out a warning telling the user how many GB of data they are about to download and unpack.  Maybe let them know that they can comment out unneeded temperatures in the script.